wren-ai-service/demo/sample_dataset/ecommerce_duckdb_mdl.json

```diff
@@ -547,165 +467,77 @@
     {
-      "name": "PaymentsOrders",
-      "models": [
-        "payments",
-        "orders"
-      ],
+      "name": "PaymentsOrderidOrdersOrderid",
+      "models": ["payments", "orders"],
       "joinType": "MANY_TO_ONE",
-      "condition": "payments.OrderId = orders.OrderId",
+      "condition": "\"payments\".OrderId = \"orders\".OrderId",
       "manySideSortKeys": [],
       "properties": {}
     }
   ],
   "enumDefinitions": [],
-  "metrics": [
-    {
-      "name": "Revenue",
-      "baseObject": "orders",
-      "dimension": [
-        {
-          "name": "PurchaseTimestamp",
-          "type": "TIMESTAMP"
-        },
-        {
-          "name": "CustomerId",
-          "type": "VARCHAR"
-        },
-        {
-          "name": "Status",
-          "type": "VARCHAR"
-        },
-        {
-          "name": "Size",
-          "type": "BIGINT"
-        },
-        {
-          "name": "IsReviewed",
-          "type": "BOOLEAN"
-        }
-      ],
-      "measure": [
-        {
-          "name": "Price",
-          "type": "DOUBLE",
-          "expression": "sum(order_items.Price)"
-        },
-        {
-          "name": "MoM_Price",
-          "type": "DOUBLE",
-          "expression": "sum(order_items.Price) - COALESCE(LAG(sum(order_items.Price)) OVER (ORDER BY date_trunc('MONTH', PurchaseTimestamp)), 0) / COALESCE(LAG(sum(order_items.Price)) OVER (ORDER BY date_trunc('MONTH', PurchaseTimestamp)), 1)"
-        },
-        {
-          "name": "NumberOfOrders",
-          "type": "BIGINT",
-          "expression": "count(OrderId)"
-        },
-        {
-          "name": "Peak",
-          "type": "DOUBLE",
-          "expression": "max(TotalValue)"
-        }
-      ],
-      "timeGrain": [
-        {
-          "name": "PurchaseTimestamp",
-          "refColumn": "PurchaseTimestamp",
-          "dateParts": [
-            "YEAR",
-            "MONTH",
-            "DAY"
-          ]
-        }
-      ]
-    }
-  ],
+  "metrics": [],
   "cumulativeMetrics": [],
-  "views": [
-    {
-      "name": "total_items_view",
-      "statement": "WITH customer_order_items AS (SELECT c.State, oi.ItemNumber FROM customers c JOIN orders o ON c.Id = o.CustomerId JOIN order_items oi ON o.OrderId = oi.OrderId) SELECT State, COUNT(ItemNumber) AS Total_Items_Purchased FROM customer_order_items GROUP BY State",
-      "properties": {
-        "question": "How many items have been purchased in orders placed by customers in each state?",
-        "summary": "The query counts the total number of items purchased by customers in each state.",
-        "viewId": "20b69c05-e9a3-4c4d-a421-06ba7e7a385a"
-      }
-    }
-  ],
-  "macros": [
-    {
-      "name": "MoM",
-      "definition": "(calculation: Expression, timestamp: Expression) => ({{ calculation }} - COALESCE(LAG({{ calculation }}) OVER (ORDER BY {{ timestamp }}), 0)) / COALESCE(LAG({{ calculation }}) OVER (ORDER BY {{ timestamp }}), 1)"
-    }
-  ],
+  "views": [],
+  "macros": [],
```

Comment:
curiously, why remove them?

Reply:
after an offline discussion with @cyyeh, we will keep the MDL file the same as the sample dataset, and add another MDL file with the views and other items section.